src/graph/ShowExecutor.cpp

```diff
@@ -151,6 +155,41 @@ void ShowExecutor::showHosts() {
             rows.emplace_back();
             rows.back().set_columns(std::move(row));
         }
+        {
+            // set sum of leader/partitions of all hosts
+            std::vector<cpp2::ColumnValue> row;
+            row.resize(6);
+
+            int32_t leaderCount = 0;
+            std::string leaders;
+            for (const auto& spaceEntry : leaderPartsCount) {
+                leaders += spaceEntry.first + ": " +
+                           folly::to<std::string>(spaceEntry.second) + ", ";
```

Comment:
Is there an extra comma at the end of the line?

Reply:
Good catch